test/lib/ansible_test/_data/sanity/import/importer.py

```diff
@@ -457,14 +488,28 @@ def capture_output(capture):
 
         with warnings.catch_warnings():
             warnings.simplefilter('error')
+            if sys.version_info[0] == 2:
+                warnings.filterwarnings(
+                    "ignore",
+                    "Python 2 is no longer supported by the Python core team. Support for it is now deprecated in cryptography,"
+                    " and will be removed in a future release.")
+                warnings.filterwarnings(
+                    "ignore",
+                    "Python 2 is no longer supported by the Python core team. Support for it is now deprecated in cryptography,"
+                    " and will be removed in the next release.")
+            if sys.version_info[:2] == (3, 5):
+                warnings.filterwarnings(
+                    "ignore", "Python 3.5 support will be dropped in the next release ofcryptography. Please upgrade your Python.")
+                warnings.filterwarnings(
+                    "ignore", "Python 3.5 support will be dropped in the next release of cryptography. Please upgrade your Python.")
```

Comment:
Purely cosmetic, but easier to read and consistent with the previous entries:

```suggestion
                warnings.filterwarnings(
                    "ignore", 
                    "Python 3.5 support will be dropped in the next release ofcryptography. Please upgrade your Python.")
                warnings.filterwarnings(
                    "ignore", 
                    "Python 3.5 support will be dropped in the next release of cryptography. Please upgrade your Python.")
```